### ISSUE TITLE
For Cycle 2

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#299]
## Summary

> _Specify the total amount of BSQ you are requesting, and the BSQ address that amount should be paid to:_

 - BSQ requested: `2900`

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommandations
- applies the 1BSQ=1USD rule (as still suggested on the slack #compensation channel)
- concerns cycle 2 (as discussed on the slack #dao channel)

### Substantive reports

- Support | https://github.com/bisq-network/roles/issues/64#issuecomment-500196749 | (BSQ requested = `2500`)
Evaluation is based and intended in line with : previous compensation requests and other support compensation requests.
Support volume decreased hopefully a bit compared to previous month.

- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = `200`)
Concerned monitored data files : https://github.com/HarryMacfinned/bisq-monitor
TL;DR: NTR, nodes were delivering coherent fees.

- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-500198948 | (BSQ requested = `200`)
TL;DR: NTR, nodes were delivering coherent prices.

NB1 : The monitoring role includes also manual sorting of false alarms, ie monitoring several Bisq specific pages + following of Tor metrics page + maillist
NB2 : All the alerts are given on the slack #bisq-monitor and #bisq-pricenode channels

## Contributions delivered
- Clearnet Bisq webpages monitoring : only some back-office alerts, reported in the #bisq-monitor and #bisq-markets channels [`0 BSQ`]
- https://github.com/bisq-network/proposals/issues/94 reduce discount on BSQ fees [`0 BSQ`]
